src/validation.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023 Helsing GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// Parsed protocol buffer definitions.
+pub mod data;
```

Comment:
I would not make this public if possible smh

Reply:
`data` sounds like inlined static data (thats why i was concerned initially)

Still this sounds like an internal repr for working with the parser. Is it absolutely required to make this public? It seems to increase the pub service drastically as it contains quite a lot datatypes